library/asyncio.po

```diff
@@ -81,34 +98,42 @@ msgid ""
 "`subprocesses <loop.subprocess_exec>`, handling :meth:`OS signals <loop."
 "add_signal_handler>`, etc;"
 msgstr ""
+"créer et gérer des :ref:`boucles d'événements <asyncio-event-loop>`, qui "
+"fournissent des bibliothèques asynchrones de :meth:`réseau <loop."
+"create_server>`, d'exécution de :meth:`subprocesses <loop.subprocess_exec>`, "
+"de gestion de :meth:`signaux système <loop.add_signal_handler>`, etc ;"
 
 #: ../Doc/library/asyncio.rst:54
 msgid ""
 "implement efficient protocols using :ref:`transports <asyncio-transports-"
 "protocols>`;"
 msgstr ""
+"implémenter des protocoles efficaces à l'aide de :ref:`transports <asyncio-"
+"transports-protocols>`;"
 
 #: ../Doc/library/asyncio.rst:57
 msgid ""
 ":ref:`bridge <asyncio-futures>` callback-based libraries and code with async/"
 "await syntax."
 msgstr ""
+":ref:`lier <asyncio-futures>` des bibliothèques basées sur les fonctions de "
+"rappel et développer avec la syntaxe *async*/*await*."
 
 #: ../Doc/library/asyncio.rst:65
 msgid "Reference"
-msgstr ""
+msgstr "Références"
```

Comment:
Ou "Documentation de référence" ?

Reply:
Ils ne l'utilisent pas dans le sens "a direction in a book or writing to some other book, passage, etc. " ?

Sommaire ?

Reply:
À moitié. Là c'est plutôt comme "sommaire" succint d'autres pages (APIs de haut et bas niveau, guides et tutoriels), pas comme les références d'un papier de recherche où on listerait les articles ou libre auquel le papier fait référence.
Je le met au singulier ?

Reply:
Je penche pour le singulier aussi :+1:

Reply:
Je penche plutôt pour sommaire. Aucune définition du mot référence ne semble convenir à l'utilisation qu'on en fait ici : https://www.larousse.fr/dictionnaires/francais/référence/67438 ou j'ai loupé quelque chose ?